katz/src/main/kotlin/katz/Id.kt

```diff
@@ -0,0 +1,31 @@
+package katz
+
+class Id<out A>(val value: A): HK<Id.F, A> {
+
+    class F
```

Comment:
could not be an object
![compiler error with object instead of class](https://cloud.githubusercontent.com/assets/1028275/24343150/4a7d15d0-12c5-11e7-82ca-319ba584281f.png)

the only viable options is use `enum class` instead of `class`